pkg/git/repository.go

```diff
@@ -155,3 +155,12 @@ func envFromAuthor(a *Author) []string {
 		sf("GIT_AUTHOR_NAME", a.Name),
 		sf("GIT_AUTHOR_EMAIL", a.Email)}
 }
+
+// DeleteCachedRepo removes the local clones from the promotino cache
```

Comment:
Nit, promotion*

Gave this PR a try now and all is good

```
(base) Adams-MBP:services aroberts$ ./services promote --from https://github.com/a-roberts/gitops-repo-testing --to https://github.com/a-roberts/staging --service service-a
2020/03/31 15:47:15 Cloning into 'gitops-repo-testing'...

2020/03/31 15:47:17 Cloning into 'staging'...

2020/03/31 15:47:23 created PR 2
(reverse-i-search)`ls': ls
(base) Adams-MBP:services aroberts$ ls -la /Users/aroberts/.promotion/cache/
total 0
drwxr-xr-x  2 aroberts  staff  64 31 Mar 15:47 .
drwxr-xr-x  3 aroberts  staff  96 31 Mar 15:46 ..
```

All good, I tried to think of any weird edge cases as I'm sure you did too - really couldn't come up with anything or break it.